ui/src/app/pages/data/concept/concept-set-details.tsx

```diff
@@ -313,6 +369,15 @@ export const ConceptSetDetails =
                   Remove concepts</Button>
           </ModalFooter>
         </Modal>}
+        {copying && <CopyModal
+          fromWorkspaceNamespace={workspace.namespace}
+          fromWorkspaceName={workspace.id}
+          fromResourceName={conceptSet.name}
+          resourceType={ResourceType.CONCEPT_SET}
+          onClose={() => this.setState({copying: false})}
+          onCopy={() => this.onCopy()}
```

Comment:
same question re: the fat arrow function vs just function name. Maybe it's a scope thing.